tcl/response.tcl

```diff
@@ -1,18 +1,30 @@
 namespace eval qc::response {
 
-    namespace export status record message action extend
+    namespace export status record message action extend data
     namespace ensemble create -unknown {
         response_subcommand_map
     }
 
+    proc data {tson} {
+        #| Sets the JSON response object data property using tson
+        global data
```

Comment:
Is this global the one Nicky already chose to write the response object to or a new global ?